Xamarin.Forms.Platform.Android/AccessibilityDelegateAutomationId.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.Views.Accessibility;
+using AndroidX.AppCompat.Widget;
+using Xamarin.Forms.Platform.Android.FastRenderers;
+using AAccessibilityDelegate = Android.Views.View.AccessibilityDelegate;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	class AccessibilityDelegateAutomationId : AAccessibilityDelegate
```

Comment:
This is very similar to what @ryl created here

https://github.com/xamarin/Xamarin.Forms/pull/5237/files#diff-d62639210acd753c36ed2128b141cb10e5bc1b4f5f1d1d95593823572f0c08f3R1